compiler/tf2tfliteV2/tf2tfliteV2.py

```diff
@@ -142,6 +143,22 @@ def _parse_array(arrays, type_fn=str):
     return list(map(type_fn, arrays.split(",")))
 
 
+def _v1_convert_keras(model_path):
+    model = tf.keras.models.load_model(model_path)
+    try_saved_model = False
+    for l in model.layers:
+        if l.name in ["lstm", "simple_rnn", "gru"]:
```

Comment:
Can you please add some comments behind this? 
The codes seems to save `keras mode` to `saved model file` and then get the converter...

Reply:
You are right, this is exactly what this code does, I tried several combinations (see list below) and at first look this PR solution looked like the only one working, so I created this PR.

Though, my methodology is a mess, (I did not check all tf versions and ways), so then I carefully analyzed results and maybe this PR is not necessary.
Looks like RNN models are properly converted only since some version of TF (like 2.1.0) by one-step converters.

What I tried:
- tf1.13.1 (checkpoint) -> freeze_graph(1.13.1 and 2.0.0) -> TFLiteConverter.from_frozen_graph (**failed** on graph freeze)
- tf2.3.1 (checkpoint) -> freeze_graph(1.13.1 and 2.0.0) -> TFLiteConverter.from_frozen_graph (**failed** on graph freeze)
- tf1.13.1 (**keras model**) -> keras load + keras save (tf 2.3.1, 2.1.0, 2.2.1) -> TFLiteConverter.from_saved_model (v2) (**SUCCESS**)
- tf1.13.1 (**keras model**) -> tflite_convert (tf 2.0.0)(v1 and v2) (**failed** with ValueError: Cannot find the Placeholder op that is an input to the ReadVariableOp.)
- tf1.13.1 (saved model) -> tflite_convert (1.13.1) (**failed** with segfault)
- tf1.13.1 (saved model) -> tflite_convert (2.0.0) (**failed** with "ValueError: Cannot find the Placeholder op that is an input to the ReadVariableOp.")
- tf1.13.1 (saved model) -> tflite_convert (2.3.1) (**failed**  with exception during exception handling)
- tf1.13.1 (saved model) -> TFLiteConverter.from_saved_model (2.3.1, toco/MLIR versions of converter) (**failed** with exception during exception handling)